### ISSUE TITLE
Sliders are not announced on Android

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#12727]
Sliders are not announced on Android. It should say something like "69 percent, brightness level, seek control, use volume keys to adjust".
